tests/Integ/CrtContext.php

```diff
@@ -0,0 +1,303 @@
+<?php
+namespace Aws\Test\Integ;
+
+use Aws\EventBridge\EventBridgeClient;
+use Aws\EventBridge\Exception\EventBridgeException;
+use Aws\Route53\Route53Client;
+use Aws\Sts\StsClient;
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Aws\S3\S3Client;
+use Aws\S3Control\S3ControlClient;
+use GuzzleHttp\Psr7;
+use PHPUnit\Framework\Assert;
+use Psr\Http\Message\StreamInterface;
+
+class CrtContext implements Context, SnippetAcceptingContext
+{
+    use IntegUtils;
+
+    private static $multiRegionAccessPoint;
+    private static $tempFile;
+    private static $eventBuses = [];
+    private static $healthCheckArn;
+    private static $healthCheckId;
+    private static $globalEndpoint;
+
+    /** @var EventBridgeClient */
+    private $eventBridgeClient;
+    /** @var array */
+    private $eventConfig;
+    /** @var S3Client */
+    private $s3Client;
+    /** @var StreamInterface */
+    private $stream;
+
+    /**
+     * @BeforeFeature @mrap
+     */
+    public static function setMultiRegionAccessPoint()
+    {
+        $sts = new StsClient([
+            'region' => 'us-east-1',
+            'version' => 'latest'
+        ]);
+        $identity = $sts->getCallerIdentity([])['Account'];
+        $s3control = new S3ControlClient([
+            'region' => 'us-west-2',
+            'version' => 'latest'
+        ]);
+        $mraps = $s3control->listMultiRegionAccessPoints([
+            'AccountId' => $identity
+        ]);
+        $alias = $mraps['AccessPoints'][0]['Alias'];
```

Comment:
Is there a better way to do this than just looking for the first item in a list call?